src/lib.rs

```diff
@@ -260,9 +374,94 @@ impl<G: Game> IsmctsHandler<G> {
     pub fn state(&self) -> &G {
         &self.root_state
     }
+
+    pub fn dotty_graph(&self) {
+        println!("{}", self.dotty_graph_string());
+    }
+
+    pub fn dotty_graph_string(&self) -> String {
+        let mut output = String::new();
+        output.push_str(
+            r#"digraph G {
+            rankdir="LR";
+            node[label=""];
+            node[shape=circle,size=100,style=filled];
+            "#,
+        );
+        self.dotty_all_children_string(&self.root_node, &mut output);
+        output.push_str("}\n");
+        output
+    }
+
+    fn get_max_visits(&self, nodes: Vec<Arc<Node<G>>>) -> usize {
```

Comment:
```suggestion
    fn max_visits_of(nodes: &[Arc<Node<G>>]) -> usize {
```
and when called pass nodes as `&nodes` instead of cloning